crates/apollo-parser/src/parser/grammar/selection.rs

```diff
@@ -47,7 +55,7 @@ pub(crate) fn selection(p: &mut Parser) {
             }
             _ => {
                 if !has_selection {
-                    p.err("exepcted at least one Selection in Selection Set");
+                    p.err("exepected at least one Selection in Selection Set");
```

Comment:
heh. this is still spelled incorrectly. 

```suggestion
                    p.err("expected at least one Selection in Selection Set");
```